snippets/elixir.snippets

```diff
@@ -197,3 +197,15 @@ snippet >i pipe to inspect
 	|> IO.inspect
 snippet >il pipe to inspect with label
 	|> IO.inspect(label: "${1:label}")
+snippet gs basic genserver structure
+	use GenServer
+
+	@doc false
+	def start_link do
+		GenServer.start_link(__MODULE__, ${1:Map.new}, name: __MODULE__)
```

Comment:
We should not default to naming the process. Naming process by default removes the ability to test them concurrently.

Reply:
True, I just don't often create unnamed processes - I'll change it.